src/frontends/onnx/frontend/src/op/com.microsoft/skip_simplified_layer_normalization.cpp

```diff
@@ -0,0 +1,136 @@
+﻿// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "core/operator_set.hpp"
+#include "exceptions.hpp"
+#include "openvino/frontend/exception.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/divide.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/range.hpp"
+#include "openvino/op/reduce_mean.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/sqrt.hpp"
+#include "utils/common.hpp"
+
+using namespace ov::op;
+using ::ONNX_NAMESPACE::TensorProto_DataType;
+
+namespace ov {
+namespace frontend {
+namespace onnx {
+namespace com_microsoft {
+namespace opset_1 {\
+
+ov::OutputVector simplified_layer_normalization(const ov::frontend::onnx::Node& node) {
+    const auto inputs = node.get_ov_inputs();
+    const auto num_inputs = inputs.size();
+
+    FRONT_END_GENERAL_CHECK(num_inputs == 2,
+                            "SimplifiedLayerNormalization takes 2 inputs. Provided " + std::to_string(num_inputs));
+
+    auto X = inputs.at(0);            // required input tensor
+    const auto scale = inputs.at(1);  // required scale tensor
+
+    CHECK_VALID_NODE(node,
+                     X.get_element_type() == ov::element::f16 || X.get_element_type() == ov::element::f32 ||
+                         X.get_element_type() == ov::element::f64 || X.get_element_type() == ov::element::bf16,
+                     "Unsupported input X type, accepted float16, float, double, bfloat16, got: ",
+                     X.get_element_type());
+    CHECK_VALID_NODE(node,
+                     scale.get_element_type() == ov::element::f16 || scale.get_element_type() == ov::element::f32 ||
+                         scale.get_element_type() == ov::element::f64 || scale.get_element_type() == ov::element::bf16,
+                     "Unsupported input scale type, accepted float16, float, double, bfloat16, got: ",
+                     scale.get_element_type());
+
+    float epsilon = node.get_attribute_value<float>("epsilon", 1e-5f);  // required
```

Comment:
Not sure I get comment "// required", if you have a default value, then it should be // optional. I use // required only in case an attribute doesn't have a default value and must throw an exception in case it isn't explicitly provided.

Reply:
Done